build/assets/tuned/openshift-node-real-time-kernel

```diff
@@ -19,4 +19,4 @@ kernel.timer_migration = 0
 /sys/devices/system/machinecheck/machinecheck*/ignore_ce = 1   
 
 [scheduler]
-isolated_cores={{.IsolatedCpus}}
+{{if .IsolatedCpus}}isolated_cores={{.IsolatedCpus}}{{end}}
```

Comment:
I think we should put the whole `[scheduler]` section under `if` condition

Reply:
And add the no_balance_cores={{. IsolatedCpus}} conditional of .enableIsolcpus (which we should rename)

Reply:
@davidvossel This was basically my only comment now. Lets keep the boolean, just change the name to something like `balance_isolated_cores` that defaults to true. And add no_balance_cores={{. IsolatedCpus}} when the balancing is disabled.

Reply:
I fixed this by calling the boolean `balanceIsolated=[true|false]`. The no_balanace_cores is something I think we should follow up on in another pr as there's probably some discussion around how this option related to isolcpu.